shared/js/background/chrome-events.es6.js

```diff
@@ -130,6 +131,11 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
         return true
     }
 
+    if (req.getExtensionVersion) {
+        res(browserWrapper.getExtensionVersion())
+        return true
+    }
```

Comment:
It's not strictly necessary to send this via the background in Chrome/FF, but I want to keep a consistent-ish API.

Reply:
👍